cactusbot/api.py

```diff
@@ -154,6 +154,12 @@ class Command(CactusAPIBucket):
             self.api.patch("/user/{token}/command/{command}/count".format(
                 token=self.api.token, command=command), data=json.dumps(data)))
 
+    async def update(self, command, value):
+        """Update command attributes."""
+
+        return await self.api.patch("/user/{token}/command/{command}".format(
+            token=self.api.token, command=command), data=json.dumps(value))
```

Comment:
No - I copied this update function from configs. Keeping it the way it is incase we ever want to update something else about the command, say the name.